src/formatter.ts

```diff
@@ -0,0 +1,130 @@
+import { dim, magenta, red, reset, underline, yellow } from 'chalk';
+import { ESLint } from 'eslint';
+import stripAnsi from 'strip-ansi';
+import table from 'text-table';
+
+function pluralize(word: string, count: number): string {
+  return count === 1 ? word : `${word}s`;
+}
+
+export function formatter(
+  results: ESLint.LintResult[],
+  shouldIncludeTodo?: boolean
+) {
+  let output = '\n',
+    errorCount = 0,
+    warningCount = 0,
+    todoCount = 0,
+    fixableErrorCount = 0,
+    fixableWarningCount = 0,
+    chalkColorFunction = yellow,
+    hasAnyErrors = false;
+
+  results.forEach((result) => {
+    const messages = result.messages;
+
+    if (messages.length === 0) {
+      return;
+    }
+
+    errorCount += result.errorCount;
+    warningCount += result.warningCount;
+    todoCount += result.todoCount | 0;
+    fixableErrorCount += result.fixableErrorCount;
+    fixableWarningCount += result.fixableWarningCount;
+
+    const areAllMessagesTodo = messages.every(
+      (message) => (message.severity as number) === -1
+    );
+
+    if (shouldIncludeTodo || !areAllMessagesTodo) {
+      output += `${underline(result.filePath)}\n`;
+    }
+
+    let messageRows: Array<Array<{}>> = [];
+    messages.forEach((message) => {
+      let messageType;
+
+      if (message.fatal || message.severity === 2) {
+        messageType = red('error');
+        hasAnyErrors = true;
+      } else if ((message.severity as number) === -1) {
+        messageType = magenta('todo');
```

Comment:
Let me know if you prefer another color (or to be consistent with template lint... I don't know if there are colors for these there).

Reply:
I think template-lint uses yellow/red for warnings/errors. I don't have a preference, but I do think a color that makes todos feel like an ambient message rather than a critical one will be useful.

Reply:
So does `magenta` work for you?

Reply:
I have no idea what magenta is :)

Reply:
It looks like it's a purple-ish color. Can we use blue instead?

Reply:
Blue it is :)